main.py

```diff
@@ -68,6 +61,14 @@ def button_click():
 
     return jsonify(result)
 
+def verify_token(token):
```

Comment:
This function shouldn't belong in main, maybe put it in Tubey and use `tubey.verify_token(<token>)`?